src/app/pages/onboarding/sign-in/utils/mnemonic-validation.ts

```diff
@@ -0,0 +1,24 @@
+import { getDefaultWordlist, wordlists } from 'bip39';
```

Comment:
I didn't think we had bip39 installed, we use https://github.com/paulmillr/scure-bip39 🤔 

can you pull list frm there?

Reply:
OK no probs, I can change it. I used that as we are using `validateMnemonic` from there

Reply:
I swapped it out and removed `bip39`